.github/workflows/update-ami.py

```diff
@@ -67,4 +73,14 @@ def parse_amichange(object):
     print(response)
   except client.exceptions.ResourceInUseException:
     print("The product is currently blocked by Amazon.  Please check the product site for more details")
+  except Exception as e:
+    print(f"An error occurred while updating AMI delivery options: {e}")
+
+def main():
+  if skip_create_ami == "True":
+    packer_build(packerfile)
+  else:
+    update_ami(amifile, packer_build_update(packerfile))
 
+if __name__ == '__main__':
+  main()
```

Comment:
I feel like it's more normal for files to end with the trailing `\n`.
```suggestion
  main()

```

Reply:
It already does, I might be confused about what you are asking?